cdap-app-fabric/src/test/java/io/cdap/cdap/internal/app/services/http/handlers/ProgramLifecycleHttpHandlerTest.java

```diff
@@ -118,6 +119,120 @@ public class ProgramLifecycleHttpHandlerTest extends AppFabricTestBase {
   private static final String STOPPED = "STOPPED";
   private static final String RUNNING = "RUNNING";
 
+  /*
+  * TODO: These tests will be modified after LCM changes to the schedules and it's apis.
```

Comment:
this test doesn't do anything with schedules, it directly starts and stops programs. It looks like it should be modified, not ignored.

Reply:
also, don't move it around in the file if you don't need to, it's hard to tell from the diff if anything in the test changed.

Reply:
Yes, it operates on a program-id. This program id is using the default -SNAPSHOT version. The apis in ProgramLifecycleHandler will need to be modified for these tests to work  and until then we ignore them. Thats what this JIRA is about https://cdap.atlassian.net/browse/CDAP-19576